index.html

```diff
@@ -0,0 +1,149 @@
+          <a href="#">Logout</a>
+          <a href="#">Mypage</a>
+          <a href="#">Spanish</a>
+        </div>
+      </div>
+      <div class="navbar toggle-selection">
+        <a class="ham-link"href="#">
+          <img class= "ham-img" src="styles-conference/assets/ham.png" alt="">
+        </a>
+        <a href="#"><img class="logo-navbar" src="styles-conference/assets/sd_logo.png" alt="logo_sd"></a>
+        <ul class="menu-list-desktop">
+          <li><a class="home-about" href="./about.html">About</a></li>
+          <li><a class="menu-item-desktop" href="#program">Program</a></li>
+          <li><a class="menu-item-desktop" href="#AJ">Join</a></li>
+          <li><a class="menu-item-desktop" href="#sponsors">Sponsor</a></li>
+          <li><a class="menu-item-desktop" href="#news">News</a></li>
+          <li><button class="sdssl-button">SDSSL Campaign</button></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="home toggle-selection">
+    <section class="welcome-page">
+      <div class="headings">
+        <h2 class="hello">"Hello! Swing Dancers"</h2>
+        <h1 class="title">SWING DANCE STYLES AND STEPS LIBRARY 2023</h1>
+      </div>
+      <p class="description">Swing dance is an umbrella term for a group of Afro-American dances that developed with the swing style of jazz music in the 1920s–1950s.</p>
+      <a class="link-sc" href="https://store.swingcity.com.ar/">
+      <div class ="sc-section">
+        <div class="sc-text">
+          <h2 class="where-to-learn">Where to learn?</h2>
+          <p class="p-link-sc">@ Swing City, school of dance.</p>
+        </div>  
+        <img class="sc-logo" src="styles-conference/assets/sc_logo.png" alt="sc_logo">
+      </div>
+      </a>
+    </section>
+  </section>
+  <section id="program" class="main-styles toggle-blur">
+    <h2 class="dance-styles">Main Dance Styles</h2>
+    <hr class="hr-red">
+    <br>
+    <ul class="styles-container">
+      <li>
+        <img class="style-icon" src="styles-conference/assets/lindy.png" alt="lindy_logo">
+        <h2>Lindy&nbspHop</h2>
+        <p>Wide range of tempos. Smooth and gentle, sometimes full of energy and crazy! (Partner)</p>
+      </li>
+      <li>
+        <img class="style-icon" src="styles-conference/assets/charleston.png" alt="lindy_logo">
+        <h2>Charleston</h2>
+        <p>
+          Energic, large and crazy moves, it gets more challenging as tempo speeds up! (Partner)</p>
+      </li>
+      <li>
+        <img class="style-icon" src="styles-conference/assets/balboa.png" alt="lindy_logo">
+        <h2>Balboa</h2>
+        <p>
+          Designed to use the minimum space possible and for faster tempos! (Partner)</p>
+      </li>
+      <li>
+        <img class="style-icon" src="styles-conference/assets/shag.png" alt="lindy_logo">
+        <h2>Shag</h2>
+        <p>
+          Quick, flashy footwork and a chugging rhythm. Very athletic and energic dance! (Partner)</p>

```

Comment:
- [OPTIONAL] Kindly make sure your list items have a hover effect similar to the Behance desing as we have to stick to it :recycle: 
![Screenshot from 2023-04-20 22-26-58](https://user-images.githubusercontent.com/97834160/233480250-30548682-60a2-4102-ae32-7937ff53496d.png)